libs/orders/src/lib/components/order-list-manager/order-list-manager.tsx

```diff
@@ -53,6 +54,15 @@ const CancelAllOrdersButton = ({
   ) : null;
 };
 
+const initialFilter: Filter = {
+  status: {
+    value: [OrderStatus.STATUS_ACTIVE, OrderStatus.STATUS_PARTIALLY_FILLED],
+  },
+  type: {
+    value: [OrderType.TYPE_LIMIT],
```

Comment:
question (blocking): Are you intentionally filtering by order type too here? Pretty sure we want all regular (lp orders aside) types showing

Reply:
It's already there. `marketId` is passed to as variable to data provider next to `filter `